src/dist/replication/meta_server/meta_bulk_load_service.cpp

```diff
@@ -1042,6 +1049,62 @@ void bulk_load_service::update_app_not_bulk_loading_on_remote_storage(
         });
 }
 
+// ThreadPool: THREAD_POOL_META_STATE
+void bulk_load_service::on_control_bulk_load(control_bulk_load_rpc rpc)
+{
+    const std::string &app_name = rpc.request().app_name;
+    const auto &control_type = rpc.request().type;
+    auto &response = rpc.response();
+    response.err = ERR_OK;
+
+    int32_t app_id;
+    {
+        zauto_read_lock l(app_lock());
+        std::shared_ptr<app_state> app = _state->get_app(app_name);
+        if (app == nullptr || app->status != app_status::AS_AVAILABLE) {
+            derror_f("app({}) is not existed or not available", app_name);
+            response.err = app == nullptr ? ERR_APP_NOT_EXIST : ERR_APP_DROPPED;
+            response.hint_msg = fmt::format("app({}) is not existed or not available", app_name);
```

Comment:
`error_code` has a function `to_string()`, you can use it to convert code to string when log and fill `hint_msg ` filed to avoid ambiguity error message.

Reply:
In shell command (in pegasus repo), I will print error code and hint_msg like `error={} [hint:\"{}\"]\n", err, hint_msg`. Although the error code is clear enough to explain the error, I still add a hint message for convenience.